packages/python/plotly/plotly/tests/test_optional/test_px/test_px_input.py

```diff
@@ -64,62 +74,100 @@ def test_with_index():
     assert fig.data[0]["hovertemplate"] == "item=%{x}<br>total_bill=%{y}<extra></extra>"
 
 
-def test_pandas_series():
-    tips = px.data.tips()
-    before_tip = tips.total_bill - tips.tip
+@pytest.mark.parametrize("constructor", constructors)
+def test_series(request, constructor):
+    if constructor is pa.table:
+        request.applymarker(pytest.mark.xfail)
+
+    data = px.data.tips().to_dict(orient="list")
+    tips = nw.from_native(constructor(data))
+    before_tip = (tips.get_column("total_bill") - tips.get_column("tip")).to_native()
+    # By converting to native, we lose the name for pyarrow chunked_array and the last
+    # assertion fails
```

Comment:
Reason for pyarrow to fail: chunked_array's are nameless, therefore hovertemplate will take another format.